wandb/apis/public/api.py

```diff
@@ -713,7 +713,7 @@ def from_path(self, path):
                 return public.BetaReport(
                     self.client,
                     {
-                        "display_name": urllib.parse.unquote(name.replace("-", " ")),
+                        "displayName": urllib.parse.unquote(name.replace("-", " ")),
```

Comment:
was this bug / does it require consumer's to update their end? seems like maybe a breaking change right?